adlfs/tests/test_spec.py

```diff
@@ -401,10 +403,10 @@ def test_time_info(storage):
     )
 
     creation_time = fs.created("data/root/d/file_with_metadata.txt")
-    assert creation_time == storage.insert_time
+    assert_almost_equal(creation_time, storage.insert_time, datetime.timedelta(seconds=1))
 
     modified_time = fs.modified("data/root/d/file_with_metadata.txt")
-    assert modified_time == storage.insert_time
+    assert_almost_equal(modified_time, storage.insert_time, datetime.timedelta(seconds=1))
```

Comment:
I saw this test flake, so I thought I would fix it.

Reply:
Looks like a small linting error. I  *think* this repo uses pre-commit, if you want to set that up locally.